src/pages/landing/landingPage.js

```diff
@@ -46,7 +47,8 @@ LandingPage.defaultProps = {
 };
 
 const mapDispatchToProps = dispatch => ({
-  manageWalkthroughServices: () => manageMiddlewareServices(dispatch),
+  // manageWalkthroughServices: () => manageMiddlewareServices(dispatch),
+  manageWalkthroughServices: () => prepareWalkthroughNamespace(dispatch, walkthroughs.oneA),
```

Comment:
I don't think we want to prepare the walkthrough namespaces here. On the landing page we want to provision the ServiceInstance namespace. We probably want to provision the Walkthrough namespace when a user enters the walkthrough itself.

Reply:
+1, but create that after the user clicks on "Getting started" we don't want to setup the workspace if the user just wants to read the description before choosing the one walkthrough to do.

Reply:
Not sure if you've resolved this already. But the approach I took is to start creating the services when the user enters the task pages instead of the tutorial pages. This ensures they've entered the steps of the tutorial.

Reply:
Agree with @aidenkeating we should prepare the walkthrough when the user enters the tutorial page.